.devcontainer/Scipy2024/environment.yml

```diff
@@ -1,20 +0,0 @@
-name: ssec-scipy2024
-channels:
-  - conda-forge
-  - defaults
-dependencies:
-  - python=3.11
-  - ipykernel
-  - ipywidgets
-  - huggingface_hub
-  - transformers<4.39
-  - ai2-olmo
-  - pytorch
-  - tqdm
-  - fsspec
-  - gcsfs
-  - pandas
-  - sentence-transformers
-  - pip
-  - pip:
-    - qdrant-client
```

Comment:
Can we have a link to this file somewhere?

Reply:
@lsetiawan I added instructions to the README for "Using a Conda Environment". As part of the directions, I give the user instructions on how to directly download the `environment.yml` file onto their local machine using `wget`.